### PR TITLE
Document validate_po.py script in readme and github PR template

### PR DESCRIPTION
**Pull request by deltragon**
See #620 and #608.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Description
+
+Describe your changes here, and link to related issues if available.
+
+Reminder to run `python validate_po.py --extract` if you have added new translatable strings.
```
Comment (archisman-panigrahi) [inline .github/pull_request_template.md:5]: Even if the person who sends the PR does not run it, the CI will run it, isn't it?

File: README.md [hunk 2]
```diff
@@ -184,6 +184,14 @@ For more details, please check the issue: [#329](https://github.com/slgobinath/S
...
+
+When adding new translatable strings in the source code, make sure to run `python validate_po.py --extract` to add them to the translation template. You will need to install `python3-polib` for this.
+
+Examples for translatable strings are `_("This is a string")` in Python code, or `<property name="label" translatable="yes">This is a label</property>` in Glade/xml files.
+
+To ensure the new strings are well-formed, you can use `python validate_po.py --validate`.
+
 ## How to Release?
 
 0. Run `update-po.sh` to generate new translation files (which will be eventually updated by translators). Commit and push the changes to the master branch.
```
[the remark this answers is not in the document]
Comment (deltragon) [reply README.md:197]: The issue is, that is going to lead to a lot of merge conflicts.
When two PRs both add translatable strings, they will both touch all .po files for all languages - and whenever Weblate translations get submitted, they also touch the .po files.
There is some potential for merge conflicts on the .pot template file as well - but there, it's only a single file, and not dozens.

[the remark this answers is not in the document]
Comment (deltragon) [reply README.md:197]: If we decide to do that anyway, I'd rather just have `validate_po.py --export` run the `update-po.sh` internally.

File: .github/pull_request_template.md (hunk 1 above)
Comment (deltragon) [reply .github/pull_request_template.md:5]: CI will complain if they didn't run it, yes. But they have to run it/add it to their change themselves.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (archisman-panigrahi) [reply README.md:197]: Alright. Let's not include `update-po.sh`. Merge conflicts are a pain. We will run `update-po.sh` on regular intervals.

**Pull request merged**